docs/user/user/metrics/validation-checklist.md

```diff
@@ -0,0 +1,30 @@
+# Validating the collected data
+
+Once instrumentation is added to a product, it is worth investing time early to understand if the data looks as expected and reasonable, while clients start sending it, and act if it does not.
+
+The following checklist could help guide such validation effort.
+
+1. Before releasing the product with the new data collection, make sure the data looks as expected by generating sample data on a local machine and inspecting it on the [Glean Debug View](../../reference/debug/debugViewTag.md):
```

Comment:
Should we link to [the debugging overview](https://mozilla.github.io/glean/book/reference/debug/index.html) instead? (( That reminds me, I do need to get to https://bugzilla.mozilla.org/show_bug.cgi?id=1762281 sooner rather than later ))